java/src/main/java/com/powsybl/python/network/NetworkCFunctions.java

```diff
@@ -677,7 +680,7 @@ public static void updateNetworkElementsExtensionsWithSeries(IsolateThread threa
                                                                  PyPowsyblApiHeader.ExceptionHandlerPointer exceptionHandlerPtr) {
         doCatch(exceptionHandlerPtr, () -> {
             String name = CTypeUtil.toString(namePtr);
-            NetworkDataframeMapper mapper = NetworkDataframes.getExtensionDataframeMapper(name);
+            NetworkDataframeMapper mapper = NetworkDataframes.getExtensionDataframeMapper(name, Optional.empty());
```

Comment:
Actually we should also adapt this "update" part so that "update_extensions" also works with the additional table argument